src/main/resources/db/migration/rdkit/postgres/V2.3.0.4__rdkit_structure.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS rdkit_structure
+(
+  id bigint NOT NULL,
+  pre_reg CHAR(40) NOT NULL,
+  reg CHAR(40) NOT NULL,
+  mol TEXT NOT NULL,
+  CONSTRAINT rdkit_structure_pkey PRIMARY KEY (id)
+);
+
+CREATE INDEX rdkit_structure_pk_idx ON rdkit_structure(id);
+CREATE INDEX rdkit_structure_pre_reg_idx ON rdkit_structure(pre_reg);
+CREATE INDEX rdkit_structure_reg_idx ON rdkit_structure(reg);
+
+
+-- LOT - Original structure/As drawn
```

Comment:
@brianbolt I think these were just notes you were taking on a call of ours. Please remove them from this migration.

Reply:
I'm actually going to delete this entire migration.  Looks like hibernate will generate this table for us.  It will also create the indexes if we give it the right annotation.

Reply:
Fixed